src/RoatpCourseManagement/README.md

```diff
@@ -0,0 +1,54 @@
+## ⛔Never push sensitive information such as client id's, secrets or keys into repositories including in the README file⛔
+
+# _Roatp Course Management Outer API_
+
+<img src="https://avatars.githubusercontent.com/u/9841374?s=200&v=4" align="right" alt="UK Government logo">
+
+[![Build Status](https://sfa-gov-uk.visualstudio.com/Digital%20Apprenticeship%20Service/_apis/build/status%2FAPIM%2Fdas-apim-endpoints-RoatpCourseManagement?repoName=SkillsFundingAgency%2Fdas-apim-endpoints&branchName=master)](https://sfa-gov-uk.visualstudio.com/Digital%20Apprenticeship%20Service/_build/latest?definitionId=2880&repoName=SkillsFundingAgency%2Fdas-apim-endpoints&branchName=master)
```

Comment:
Use this link

[![Build Status](https://sfa-gov-uk.visualstudio.com/Digital%20Apprenticeship%20Service/_apis/build/status%2FAPIM%2Fdas-apim-endpoints-RoatpCourseManagement?repoName=SkillsFundingAgency%2Fdas-apim-endpoints&branchName=refs%2Fpull%2F1374%2Fmerge)](https://sfa-gov-uk.visualstudio.com/Digital%20Apprenticeship%20Service/_build/latest?definitionId=2880&repoName=SkillsFundingAgency%2Fdas-apim-endpoints&branchName=refs%2Fpull%2F1374%2Fmerge)

Reply:
There is a package which is now throwing a warning, hence partially succeeded above